doc/aggregation.md

```diff
@@ -0,0 +1,164 @@
+# Enable ClickHouse aggregation
+
+It's enabled by default since [#157](https://github.com/lomik/graphite-clickhouse/pull/157). You can disable it to use aggregation in graphite-clickhouse.
+
+```
+[clickhouse]
+# ClickHouse-side aggregation
+internal-aggregation = true
+# maximum number of points per metric. It should be set to 4096 or less for ClickHouse older than 20.8
+# https://github.com/ClickHouse/ClickHouse/commit/d7871f3976e4f066c6efb419db22725f476fd9fa
+max-data-points = 1048576
+```
+
+The only known _frontend_ supporting passing `maxDataPoints` from requests is [carbonapi>=0.14](https://github.com/go-graphite/carbonapi/releases/tag/0.14.0). The used protocol should be `carbonapi_v3_pb`, see [config](https://github.com/go-graphite/carbonapi/blob/main/doc/configuration.md#upstreams)->backendv2->backends->protocol.
```

Comment:
`The used protocol should be` maybe better to say `Protocol should be set to carbonapi_v3_pb for this feature to fully work`

Reply:
done